CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Misc:
 Fixes:
 
 - [#2398](https://github.com/rails-api/active_model_serializers/pull/2398) Update rails dependency to < 6.2 (@ritikesh)
+- [#2399](https://github.com/rails-api/active_model_serializers/pull/2399) Handles edge case where requested current_page > total_pages (@f3z0)
```

Comment:
as 0.10.12 has already been published, I guess this change would fit under Unreleased -> Fixes section.

Reply:
Sorry about that, this makes more sense.